code/datums/martial/sleeping_carp.dm

```diff
@@ -70,7 +51,6 @@
 	return
 
 /datum/martial_art/the_sleeping_carp/grab_act(mob/living/A, mob/living/D)
-	add_to_streak("G",D)
 	if(check_streak(A,D))
 		return TRUE
```

Comment:
You might want to delete lines 54 and 55 since you're not going to be performing ```check_streak``` on grab and there is nothing to return.

Reply:
Thanks for pointing it out, I'll address it soon.